Tests/Algorithm/AlgorithmTradingTests.cs

```diff
@@ -91,7 +102,7 @@ public void SetHoldings_ZeroToShort(decimal leverage)
         public void SetHoldings_ZeroToShort_SmallConstantFeeStructure(decimal leverage)
         {
             Security msft;
-            var algo = GetAlgorithm(out msft, 1, 1);
+            var algo = GetAlgorithm(out msft, leverage, 1);
```

Comment:
it's nice that these tests are set up using `TestCaseSource` but the way they're set up implies I can add random test cases and they will work. I don't think this is the case however. If we're going to use `TestCaseSource` then the test case needs to include the answer or the test method should manually compute it using the inputs